### PR TITLE
fixed bug of WebSocket that it can not receive over 4096 byte

### PR DESCRIPTION
**Pull request by HoGarfield**

### CODE REVIEW

**Comment by dualface** [comment]
thanks HoGarfield

**Comment by edwardzhou** [comment]
this patch only fix websocket cannot receive 4096+ data. but still has problem on sending 4096+ data.

File: cocos/network/WebSocket.cpp [hunk 1]
```diff
@@ -221,6 +221,9 @@ enum WS_MSG {
 , _delegate(nullptr)
 , _SSLConnection(0)
 , _wsProtocols(nullptr)
+,_pending_frame_data_len(0)
+,_current_data_len(0)
+,_current_data(NULL)
```
Comment (minggo) [inline cocos/network/WebSocket.cpp:226]: Please use `_currentData` instead to obey [cocos2d-x coding style].
And please use `nullptr` instead. We use c++11 in develop branch.

Thanks.

**Comment by edwardzhou** [comment]
can please also include the send 4096+ bytes fix as well, ref: https://github.com/cocos2d/cocos2d-x/pull/4014

**Comment by dumganhar** [comment]
Fixed at https://github.com/cocos2d/cocos2d-x/pull/5008. Thanks.

**Pull request closed without merging**